rollup.config.mjs

```diff
@@ -1,20 +1,21 @@
-import { createRequire } from 'node:module';
+import fs from 'node:fs/promises'
 import pluginSass from './rollup.plugin.sass.mjs';
+import typescript from '@rollup/plugin-typescript';
 import rollupResolve from '@rollup/plugin-node-resolve';
 
 // require `package.json` in order to use its 'main' and 'module' fields to tell rollup where to output the generated
 // bundles
```

Comment:
Non-blocking comment - can we remove this JS comment here? It was originally put there to specify why we were doing this funky `createRequire` statement